test/grouped_convnd_bwd_weight/test_grouped_convnd_bwd_weight.cpp

```diff
@@ -27,28 +29,59 @@ class TestGroupedConvndBwdWeight : public ::testing::Test
     using NDimSpatial = std::tuple_element_t<6, Tuple>;
 
     std::vector<ck::utils::conv::ConvParam> conv_params;
-    ck::index_t split_k{2};
+    std::vector<ck::index_t> split_ks{1, 2};
+
+    bool skip_case(const ck::utils::conv::ConvParam& params, const ck::index_t split_k)
+    {
+        // K=1 or C=1 is supported only by dl kernel
+        // dl kernel is only supported for split_k=1
+        if constexpr(std::is_same_v<InDataType, ck::half_t>)
```

Comment:
If the datatype is sth else than fp16, is your comment still true? If not, maybe would be better to update it to not cause any confusion?

Reply:
Fixed comment. It only applies for fp16